### ISSUE TITLE
Training results on ImageNet and Cityscapes

### ISSUE THREAD

**Issue by Asthestarsfalll** [YihengZhang-CV/LPS-Net#6]
I trained the lpsnet-m on ImageNet 1k for 100 epoch with batch size 1024.The detials are as follows:
Optimizer:
![image](https://user-images.githubusercontent.com/72954905/202892919-0b7cfaad-d72d-42fc-868d-96f8211f8532.png)
Data augments:
![image](https://user-images.githubusercontent.com/72954905/202892939-00f8c45c-b7f6-4ae8-8565-68e24e600973.png)
Eval dataset:
![image](https://user-images.githubusercontent.com/72954905/202893004-d25a52bb-c436-4051-b973-307bf56aa3bb.png)
result:
![image](https://user-images.githubusercontent.com/72954905/202893060-a32ed4c4-7f61-492f-8dda-b033152221e4.png)

Then I use the pre-trained model to train a lpsnet-m on cityscapes for 180k iterations with batch size 16. The detials are as follows:
Loss function:
![image](https://user-images.githubusercontent.com/72954905/202893242-df498dab-e9df-45d9-a9f6-bfca964409ca.png)
lr_scheduler:
![image](https://user-images.githubusercontent.com/72954905/202893272-b191c3db-02db-4edd-a429-a5c9eee8938f.png)
Optimizer:
![image](https://user-images.githubusercontent.com/72954905/202893362-9e9cb5d1-e5db-4cd5-9f6d-e01aa09cf661.png)
Data augments:
![image](https://user-images.githubusercontent.com/72954905/202893377-026bc158-61e6-4d9a-9164-f700e0991013.png)
Eval dataset:
![image](https://user-images.githubusercontent.com/72954905/202893412-215877ed-ff97-45c2-8b8a-259d8b5aeca4.png)
Result:
![image](https://user-images.githubusercontent.com/72954905/202893538-0d4789d5-5826-4c19-b494-a4c763f00462.png)

**Comment by Asthestarsfalll** [comment]
Could you please tell me more details about training settings?
And did you finetune model after fusing CONV and BN?
